lab-stephen/.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
**TODO FOR LOST POINTS**

You need to include a ``.babelrc`` per instructions.